arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-congo.dts

```diff
@@ -449,8 +449,8 @@
 &i3c4 {
 	status = "okay";
 	initial-role = "primary";
```

Comment:
Please split to machine specific commits , that is the process we follows.

This commit also changes frequency , please update commit message with this details